### PR TITLE
fix: make profile picture layout responsive

### LINKED ISSUE #13: Improve profile picture layout
At the moment, the profile pictures at the top, overflow when the screen is too narrow, or there's too many pictures!

This issue will need you to go through the CSS and make sure the website is responsive!

Screenshot 2021-09-30 at 14 21 19
Screenshot 2021-09-30 at 14 21 58

### PR DESCRIPTION
**Pull request by CIPHERTron**
This PR fixes #13 
@VaibhaveS and I have collaborated on fixing this issue.
- I've worked on fixing the CSS of the tablet view
- @VaibhaveS has worked on fixing the CSS of the mobile view
- What we did in order to fix it?
   - We used CSS grid and media queries for multiple breakpoints
   - Added some extra padding at the bottom of the pod name of `h1` type

### Attached Screenshots
Screenshot 2022-01-11 at 8 20 56 PM

---

Screenshot 2022-01-11 at 8 21 17 PM

---

Screenshot 2022-01-11 at 8 22 18 PM

## Please, go through these steps before you submit a PR.

- [x] My Pod Leader knows I'm working on this Pull Request
- [X] I've explained what the Pull Request is adding.
- [X] I've explained why this is important.

### CODE REVIEW

File: _sass/profile.scss [hunk 1]
```diff
@@ -1,26 +1,39 @@
...
+
+	img {
+		border-radius: 50%;
+		max-width: 100%;
+		max-height: 100%;
+		margin-left: auto;
+		margin-right: auto;
+		display: inline;
+		padding: 20px;
+	}
```
Comment (gmelodie) [inline _sass/profile.scss:21]: Maybe trying to use 64-spaced tabs?

File: _sass/profile.scss [hunk 2]
```diff
@@ -1,26 +1,39 @@
...
+		@media (max-width: 574px) {
+			grid-template-columns: repeat(2, 1fr);
+		}
+		@media (max-width: 344px) {
+			grid-template-columns: repeat(1, 1fr);
+		}
+		margin-bottom: 30px;
+		height: fit-content;
+	}
 }
```
Comment (Aniket762) [inline _sass/profile.scss:39]: Looks Perfect!

**Review by Aniket762 — APPROVED** [review]

File: _sass/profile.scss (hunk 1 above)
Comment (yyjlincoln) [reply _sass/profile.scss:21]: Hell yeah! 64-spaced tabs are the best

**Review by yyjlincoln — APPROVED** [review]
Perfect fix!

**Review by yyjlincoln — CHANGES_REQUESTED** [review]
Hey @CIPHERTron, I would like to merge it into `main`; however, it seems that the images are too big on desktops (especially if you have a big screen, the images are taking more than half of my screen). Can you please restrict the maximum size of the images?

[Here](https://github.com/yyjlincoln/_github_issues_assets/raw/master/0.png)'s a screenshot of the issue.

**Comment by CIPHERTron** [comment]
> Hey @CIPHERTron, I would like to merge it into `main`; however, it seems that the images are too big on desktops (especially if you have a big screen, the images are taking more than half of my screen). Can you please restrict the maximum size of the images?
> 
> [Here](https://github.com/yyjlincoln/_github_issues_assets/raw/master/0.png)'s a screenshot of the issue.

Sure. Will do!

**Comment by CIPHERTron** [comment]
@yyjlincoln do we require the screenshot that you've added to the repo? Since we're done with the issue, I think we can remove it 🤔

**Comment by yyjlincoln** [comment]
> @yyjlincoln do we require the screenshot that you've added to the repo? Since we're done with the issue, I think we can remove it 🤔

Nope, we don't need that screenshot anymore. I didn't add it to the repo - that screenshot is actually hosted under my repo, `yyjlincoln/_github_issues_assets` and I will remove it shortly. Thanks though.

**Comment by yyjlincoln** [comment]
@CIPHERTron The issue is still there... Do you want to double-check?
I tried cloning your repo/netlify preview and neither of them worked.

**Comment by CIPHERTron** [comment]
@yyjlincoln 
Earlier it was taking 100% width and height of the parent div but now it only takes 93%. 90% will also look okay but reducing it further will increase the grid gap and will also look very small


Screenshot 2022-01-13 at 9 31 32 AM

**Comment by yyjlincoln** [comment]
Interesting... I will give it a try

**Comment by CIPHERTron** [comment]
> Interesting... I will give it a try. The previous commit made no difference on my side

Maybe it's your cache 😅 
Try again with a hard reload. If that doesn't work, I'll look into it.

Also, should we change the profile images from 4 in a row to 5 in a row?

**Comment by yyjlincoln** [comment]
I think a better way to do it is through `flex` which automatically goes to the next line when it overflows (instead of defining `@media` tags which may be incompatible on larger screens unless you define a larger `@media` tag). I'll submit a commit and we can discuss.

**Comment by CIPHERTron** [comment]
> I think a better way to do it is through `flex` which automatically goes to the next line when it overflows (instead of defining `@media` tags which may be incompatible on larger screens unless you define a larger `@media` tag). I'll submit a commit and we can discuss.

Oh, in that case, we can use the concept of `minmax`
We won't have to write media queries there

**Comment by yyjlincoln** [comment]
@CIPHERTron I've pushed it to the repo. What do you think?

**Comment by CIPHERTron** [comment]
> @CIPHERTron I've pushed it to the repo. What do you think? Is it working on your side?
LGTM 🎉

**Review by yyjlincoln — APPROVED** [review]
Nice!

**Comment by yyjlincoln** [comment]
> Maybe it's your cache 😅

Just saw that... It wasn't as I cloned your repo in a separate folder and made sure I cleared all caches & opened in incognito mode. Anyway - it's now fixed.

**Comment by yyjlincoln** [comment]
Merging into main as:
- At least two approvals
- Fixed outstanding issues raised
- Passing all checks.

**Pull request merged**